cocos/platform/ios/CCDirectorCaller-ios.mm

```diff
@@ -137,7 +138,10 @@ -(void) doCaller: (id) sender
         
         [EAGLContext setCurrentContext: cocos2dxContext];
 
-        CFTimeInterval dt = ((CADisplayLink*)displayLink).timestamp - lastDisplayTime;
+        CFTimeInterval dt = 0;
+        if (lastDisplayTime != 0) {
+            dt = ((CADisplayLink*)displayLink).timestamp - lastDisplayTime;
+        }
```

Comment:
I understand that, but in this case lastDisplayTime should be inited not with 1/60 but with some real timestamp, shouldn't it?